client_with_ssl.cpp

```diff
@@ -1,47 +1,81 @@
-#include <hazelcast/client/hazelcast_client.h>
 #include <string>
+#include "model/city.h"
 
+/**
+* This is boilerplate application that configures client to connect Hazelcast
+* Cloud cluster.
+* <p>
+* See: <a href="https://docs.hazelcast.com/cloud/cpp-client">https://docs.hazelcast.com/cloud/cpp-client</a>
+*/
+
+void map_example(hazelcast::client::hazelcast_client client);
+void non_stop_map_example(hazelcast::client::hazelcast_client client);
 int main(int argc, char **argv) {
     hazelcast::client::client_config config;
-    
     config.set_cluster_name("YOUR_CLUSTER_NAME");
     config.set_property(hazelcast::client::client_properties::CLOUD_URL_BASE, "YOUR_DISCOVERY_URL");
     config.set_property("hazelcast.client.statistics.enabled", "true");
-
     auto &cloud_configuration = config.get_network_config().get_cloud_config();
     cloud_configuration.enabled = true;
     cloud_configuration.discovery_token = "YOUR_CLUSTER_DISCOVERY_TOKEN";
-
     boost::asio::ssl::context ctx(boost::asio::ssl::context::tlsv12);
-    ctx.set_verify_mode(boost::asio::ssl::verify_peer);
     ctx.load_verify_file("ca.pem");
     ctx.use_certificate_file("cert.pem", boost::asio::ssl::context::pem);
     ctx.set_password_callback([&] (std::size_t max_length, boost::asio::ssl::context::password_purpose purpose) {
         return "YOUR_SSL_PASSWORD";
     });
     ctx.use_private_key_file("key.pem", boost::asio::ssl::context::pem);
```

Comment:
why do we need these lines? see tls/ssl basic example [here](https://github.com/hazelcast/hazelcast-cpp-client/blob/master/examples/tls/BasicTLSClient.cpp#L31)